includes/admin.inc

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * @file
+ * Admin form.
+ */
+
+/**
+ * Admin settings form for Islandora Simple Workflow configuration.
+ *
+ * @param array $form
+ *   An array representing a form within Drupal.
+ * @param array $form_state
+ *   An array containing the Drupal form state.
+ */
+function islandora_simple_workflow_admin_form(array $form, array &$form_state) {
```

Comment:
Wonder if we reeaaaaaally need another toggle. Can this be just shown by default??